git-add--interactive.perl

```diff
@@ -1541,7 +1541,7 @@ sub patch_update_file {
 		for (@{$hunk[$ix]{DISPLAY}}) {
```

Comment:
Hi

> It would probably make sense to use `.` rather than `,`

I see. So, it would be `print colored $prompt_color, "(". ($ix+1) ."/$num) ", ...` or do I need to replace any of the other 2 commas with period? I tried a REPL, but didn't observe any difference in the small examples I ran. 

> Could you move ... the explanation ...to the commit message?

Sure, I'll send an updated patch based on your response to the above query.

Thanks
Kunal Tyagi

Reply:
> it would be `print colored $prompt_color, "(". ($ix+1) ."/$num) ", ...` or do I need to replace any of the other 2 commas with period?

That should be fine, I think.

I probably shouldn't even have said anything because it does not really matter in the long run: I want to port your patch into the C version as soon as it is on its way into `next`.

Note: the Git mailing list did not see your reply here, nor my reply to your reply here, due to shortcomings in GitGitGadget (it currently only mirrors replies on the Git mailing list to the PR, not vice versa).

Reply:
> only mirrors replies on the Git mailing list to the PR

I see. I did wonder why I didn't see that